definitions.yaml

```diff
@@ -86,6 +103,9 @@ definitions:
         description: |
           The [release](https://docs.giantswarm.io/api/#tag/releases) version
           to use in the new cluster
+      availability_zones:
+        description: Number of availability zones a cluster should be spread across.
```

Comment:
Can we inform here that a default will be applied (as given in info endpoint -> `general. availability_zones.default`)?